sonarts-sq-plugin/sonar-typescript-plugin/src/test/java/org/sonar/plugin/typescript/ExternalTypescriptSensorTest.java

```diff
@@ -99,39 +90,12 @@ public void should_run_processes_and_save_data() {
     assertThat(issue.flows()).hasSize(1);
     assertThat(issue.gap()).isEqualTo(42);
 
-    assertThat(sensorContext.highlightingTypeAt(testInputFile.key(), 2, 3)).containsExactly(TypeOfText.KEYWORD);
-
-    assertThat(sensorContext.measure(testInputFile.key(), CoreMetrics.NCLOC).value()).isEqualTo(3);
-    assertThat(sensorContext.measure(testInputFile.key(), CoreMetrics.COMMENT_LINES).value()).isEqualTo(2);
-
-    verify(fileLinesContext).setIntValue(CoreMetrics.NCLOC_DATA_KEY, 55, 1);
-    verify(fileLinesContext).setIntValue(CoreMetrics.NCLOC_DATA_KEY, 77, 1);
-    verify(fileLinesContext).setIntValue(CoreMetrics.NCLOC_DATA_KEY, 99, 1);
-    verify(fileLinesContext).setIntValue(CoreMetrics.EXECUTABLE_LINES_DATA_KEY, 5, 1);
-    verify(fileLinesContext).setIntValue(CoreMetrics.EXECUTABLE_LINES_DATA_KEY, 7, 1);
-    verify(fileLinesContext).save();
-    verifyNoMoreInteractions(fileLinesContext);
-
-    verify(noSonarFilter).noSonarInFile(eq(testInputFile), eq(Sets.newHashSet(24)));
-
-    assertThat(sensorContext.measure(testInputFile.key(), CoreMetrics.STATEMENTS).value()).isEqualTo(100);
-    assertThat(sensorContext.measure(testInputFile.key(), CoreMetrics.FUNCTIONS).value()).isEqualTo(10);
-    assertThat(sensorContext.measure(testInputFile.key(), CoreMetrics.CLASSES).value()).isEqualTo(1);
-    assertThat(sensorContext.measure(testInputFile.key(), CoreMetrics.COMPLEXITY).value()).isEqualTo(42);
-    assertThat(sensorContext.measure(testInputFile.key(), CoreMetrics.COGNITIVE_COMPLEXITY).value()).isEqualTo(22);
-
-    List<TokensLine> cpd = sensorContext.cpdTokens(testInputFile.key());
-    assertThat(cpd).hasSize(1);
-    assertThat(cpd.get(0).getStartLine()).isEqualTo(2);
-    assertThat(cpd.get(0).getValue()).isEqualTo("foobar");
-
-    Collection<TextRange> usages = sensorContext.referencesForSymbolAt(testInputFile.key(), 2, 0);
-    assertThat(usages).hasSize(1);
-    assertThat(usages.iterator().next().start().lineOffset()).isEqualTo(6);
 
     assertThat(logTester.logs()).contains(String.format("Setting 'NODE_PATH' to %s%sfoo%snode_modules", BASE_DIR.getAbsoluteFile(), File.separator, File.separator));
   }
 
+  // only rule using forced message was no-eval S1523 which is not provided by SonarTS anymore
+  @Ignore
   @Test
   public void should_replace_message_when_forced_message_provided() {
```

Comment:
nice feature! I forgot about it. We should probably do something like it in SonarJS.

Reply:
if there is only one rule to benefit from it, I would prefer not to add complexity